### PR TITLE
Release drafter

### PR DESCRIPTION
**Pull request by norlandrhagen**

### CODE REVIEW

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+version-resolver:
+  major:
+    labels:
+      - 'major'
+  minor:
+    labels:
+      - 'minor'
+  default: patch
+exclude-labels:
+  - 'skip-changelog'
```
Comment (maxrjones) [inline .github/release-drafter.yml:22]:
```suggestion
  - 'skip-changelog'
  - 'dependencies'
```

File: .github/workflows/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  contents: read
+
+jobs:
+  update_release_draft:
+    permissions:
+      # write permission is required to create a github release
+      contents: write
+      # write permission is required for autolabeler
+      # otherwise, read permission is required at least
+      pull-requests: write
```
Comment (maxrjones) [inline .github/workflows/release-drafter.yml:19]:
```suggestion
      pull-requests: read
```
since the autolabeler isn't active

**Review by maxrjones — APPROVED** [review]
This looks good, thanks @norlandrhagen!

Since the release-drafter doesn't currently support days in the name or tag template (https://github.com/release-drafter/release-drafter/issues/936), you'll need to take caution when publishing releases using the auto-drafted notes because the title/tag will need to be changed from the default semantic versioning. Up to you if you think this is a deal-breaker for this tools usefulness for this project.

**Comment by norlandrhagen** [comment]
Closed for now in favor of manual release drafting

**Pull request closed without merging**